### ISSUE TITLE
Unit/regression tests

### ISSUE THREAD

**Issue by woodruffw** [trailofbits/mishegos#7]
It'd be nice to have some basic tests in the CI, probably using `mishegos`'s manual mode to feed some simple inputs in.

**Comment by woodruffw** [comment]
To elaborate: we probably want simple _valid_ instruction samples to test with, just to ensure that every decoder reports correctly. Even just some NOPs and simple MOVs would probably suffice.

**Comment by ekilmer** [comment]
This would be nice to catch simple runtime issues like missing symbols, even if the workers seem to build correctly (relevant PR for bddisasm https://github.com/bitdefender/bddisasm/pull/53, but this could have easily been any other tool)

**Comment by woodruffw** [comment]
Yep! It should be *very* easy to write a basic self-test, since it's just:

```
M=1 mishegos ./workers.spec <<< "90" > /dev/null
```

...or something similar, where `90` is the hex string for the instruction to test all of the workers against.
